### DISCUSSION TITLE
Tweet preview when viewing media

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#602] [Ideas]
After using harpy for scrolling through media, i noticed something which is that sometimes media have context which is in the tweet and of course you can press show and see the tweet.

But I have a better idea to implement and it's similar to another client app for reddit named "Boost" .

![20220313_010506.jpg](https://user-images.githubusercontent.com/83197825/158036684-74b7e53b-d583-4a35-9289-d3ad23b62284.jpg)
_Boost implementation_

You could put the text preview below the picture and make it have the same trigger as the "show" button we currently have, so if you want to see the tweet full screen you just tap on text, if you don't? You just got your context and you can keep scrolling no problem. 

![20220313_010434.jpg](https://user-images.githubusercontent.com/83197825/158036668-33b359bc-5cb7-4fa0-bd4a-75bcdc6a5a93.jpg)
_Example of placement_

And if I'm not wrong i remember twitter having something similar but they removed it for some reason...

**Comment by robertodoering** [comment]
Looks pretty good, thanks for the suggestion

I limited it to 2 lines which seems reasonable.
The change will go live with #587

<img width=350 src="https://user-images.githubusercontent.com/20045287/158055057-e896d9c8-43d1-4169-b340-6865b458d32d.png"/> <img width=350 src="https://user-images.githubusercontent.com/20045287/158054933-4585a351-f22e-4e78-94fc-bed040060ca5.png"/>

**Reply by FerasK0** [reply]
Great!
looks awesome.
